pkg/cluster/describe.go

```diff
@@ -345,3 +346,17 @@ func findHyperShiftMgmtSvcClusters(conn *sdk.Connection, cluster *cmv1.Cluster)
 	// Shouldn't normally happen as every management cluster should have a service cluster
 	return mgmtClusterName, ""
 }
+
+func PrintClusterWarnings(connection *sdk.Connection, cluster *cmv1.Cluster) error {
+	serviceLogs, err := connection.ServiceLogs().V1().Clusters().ClusterLogs().List().ClusterID(cluster.ID()).Send()
+	if err != nil {
+		return err
+	}
+	serviceLogs.Items().Each(func(entry *slv1.LogEntry) bool {
+		if entry.Severity() == slv1.SeverityWarning {
```

Comment:
I think we have other warnings we currently send as service logs. What if the customer would like to not be bothered about it :thinking: . Should this be behind some flag within describe? like --warnings

Reply:
We do send other warnings as service logs, but none of them come through immediately after installation. I agree that constantly receiving warnings would be a bad user experience. Ultimately it is a BU decision whether warnings get displayed or not. I think they should be displayed by default, with the policy that if the user is seeing the warning, then it describes an action that the user can take to get rid of the warning.  In the one shown here, the user can alleviate it by enabling the Org Policy API.

If it helps push this PR forward, I can add a --quiet flag for users that do not want to see any warnings. We can also filter the service logs to only show warnings of type "Cluster Lifecycle" if needed. Please let me know.